docker/standalone/dashboard-entrypoint.sh

```diff
@@ -40,7 +40,7 @@ else
   export AUTH_SECRET=$(uuidgen)
 fi
 
-export HOSTNAME=localhost
+export HOSTNAME=0.0.0.0
```

Comment:
For some reason I don't seem to understand, `localhost` was not working and thus we were not able to stablish a connection to the dashboard on the `lh-standalone` image